src/manifest.json

```diff
@@ -89,4 +128,4 @@
       "run.js"
     ]
   }
-}
+}
```

Comment:
Did prettier remove the newline at the end? That's... odd.

Reply:
(my VS-code might have some funky settings....)